controllers/flinkcluster_converter.go

```diff
@@ -106,10 +106,9 @@ func getDesiredJobManagerDeployment(
 		ports = append(ports, corev1.ContainerPort{Name: port.Name, ContainerPort: port.ContainerPort, Protocol: corev1.Protocol(port.Protocol)})
 	}
 	var jobManagerDeploymentName = getJobManagerDeploymentName(clusterName)
-	var podLabels = getLabels(*flinkCluster, map[string]string{
-		"component": "jobmanager",
-	})
-	var labels = getLabelsRevisionHashAttached(flinkCluster.Status, podLabels)
+	var componentLabels = getComponentLabels(*flinkCluster, "jobmanager")
+	var podLabels = mergeLabels(componentLabels, nil)
```

Comment:
Seems there is no need to introduce `componentLabels`, we can just use `podLabels`.

Reply:
Done.